mapie/conformity_scores/conformity_scores.py

```diff
@@ -63,47 +69,57 @@ def get_signed_conformity_scores(
 
         Parameters
         ----------
-        y: NDArray
-            Observed values.
+        X: ArrayLike
+            Observed feature values.
 
-        y_pred: NDArray
-            Predicted values.
+        y: ArrayLike
+            Observed target values.
+
+        y_pred: ArrayLike
+            Predicted target values.
 
         Returns
         -------
         NDArray
-            Unsigned conformity scores.
+            Signed conformity scores.
         """
 
     @abstractmethod
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike
     ) -> NDArray:
         """
-        Placeholder for ``get_estimation_distribution``.
+        Placeholder for ``get_signed_conformity_scores``.
         Subclasses should implement this method!
 
         Compute samples of the estimation distribution from the predicted
-        values and the conformity scores.
+        targets and ``values``that can be either the conformity scores or
+        the conformity scores aggregated with the predictions.
 
         Parameters
         ----------
-        y_pred: NDArray
-            Predicted values.
+        X: ArrayLike
+            Observed feature values.
 
-        conformity_scores: NDArray
-            Conformity scores.
+        y_pred: ArrayLike
+            Predicted values, it can be any type of predictions
+            (multi, low, up, ...).
```

Comment:
UPD: prefer to use `Predicted reference values of shape (n_samples, ...). The last dimension is the reference of the prediction.'